rimage/transform/cam_poses.go

```diff
@@ -79,8 +79,8 @@ func GetPossibleCameraPoses(essMat *mat.Dense) ([]*mat.Dense, error) {
 	poses[3].Augment(R2, &tOpp)
 	// adjust sign of poses
 	posesOut := make([]*mat.Dense, 4)
-	for i, pose := range poses {
-		posesOut[i] = mat.DenseCopyOf(adjustPoseSign(&pose))
+	for i := range poses {
+		posesOut[i] = mat.DenseCopyOf(adjustPoseSign(&poses[i]))
```

Comment:
This was the gosec G601 issue. It was probably safe to ignore in this specific case, as copies are being made, but fixing it let me enable the check again so we don't have to worry about actual bugs from this elsewhere.

Reply:
@bhaney Oh, and this one too? It passes tests and should change nothing, but just in case I'm missing something.